### PR TITLE
CATO-3974: Adding delete confirmation for 'Tangible assets' page.

### PR DESCRIPTION
**Pull request by tmikus**

### CODE REVIEW

File: src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/abridged/AC125.scala [hunk 1]
```diff
@@ -26,12 +26,36 @@ case class AC125(value: Option[Int]) extends CtBoxIdentifier(name = "The cost of
...
+      ac5131().value,
+      ac219().value,
+      ac130().value,
+      ac214().value
+    )
+
+    (values.exists(_.nonEmpty) || ac5133().value.getOrElse("").trim().nonEmpty) match {
+      case true => Set(CtValidation(None, "error.balanceSheet.tangibleAssetsNote.cannot.exist"))
+      case _ => Set.empty
+    }
```
Comment (pncampbell) [inline src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/abridged/AC125.scala]: What about a good old fashioned if/else ? :-) (unless this is not a pure boolean test and I'm missing something ...)

Comment (tmikus) [inline src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/abridged/AC125.scala]: Is this the first time a Scala dev complains about code being too Scala-like?

Comment (pncampbell) [inline src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/abridged/AC125.scala]: Theres nothing "un scala" about doing an if on a boolean expression

Comment (tmikus) [inline src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/abridged/AC125.scala]: There's no way to please you guys :joy:

**Pull request merged**